datatransfer/impl/graphsync/graphsync.go

```diff
@@ -2,326 +2,15 @@ package graphsyncimpl
 
 import (
 	"bytes"
-	"context"
-	"fmt"
-	"reflect"
 
-	"github.com/ipfs/go-cid"
-	"github.com/ipfs/go-graphsync"
 	logging "github.com/ipfs/go-log"
 	"github.com/ipld/go-ipld-prime"
 	"github.com/ipld/go-ipld-prime/encoding/dagcbor"
 	ipldfree "github.com/ipld/go-ipld-prime/impl/free"
-	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
-	"github.com/libp2p/go-libp2p-core/host"
-	"github.com/libp2p/go-libp2p-core/peer"
-
-	"github.com/filecoin-project/lotus/datatransfer"
-	"github.com/filecoin-project/lotus/datatransfer/message"
-	"github.com/filecoin-project/lotus/datatransfer/network"
 )
 
 var log = logging.Logger("graphsync-impl")
```

Comment:
as far as I can tell this just has the nodeAsBytes function now. So I would rename it to something like utils.go